pkg/volume/util/atomic_writer.go

```diff
@@ -256,7 +255,7 @@ func validatePath(targetPath string) error {
 	if targetPath == "" {
 		return fmt.Errorf("invalid path: must not be empty: %q", targetPath)
 	}
-	if path.IsAbs(targetPath) {
+	if filepath.IsAbs(targetPath) {
```

Comment:
does this break on windows when presented with paths specified in the API, which will always started with `/`?